01 Arrays & Vectors/08 subarray_sort.cpp

```diff
@@ -16,6 +16,8 @@ bool outOfOrder(vector<int> arr,int i){
 }
 
 pair<int,int> subarraySort(vector<int> arr) {
+	
+	if(arr.size() == 1) return 0;
```

Comment:
return 0 will give an error I guess here since the return type is pair<int,int>

Reply:
Yeah! You are right it should return {-1, -1} or (-1, -1) or something invalid pair I think.